src/commands/commands.js

```diff
@@ -3,21 +3,20 @@
  * See LICENSE in the project root for license information.
  */
 
-var config;
 var configEvent;
+var config;
```

Comment:
Why did you change the order of the variables?

Reply:
;)